app/server-utils/stampy.ts

```diff
@@ -263,6 +288,10 @@ export const loadQuestionDetail = withCache('questionDetail', async (question: s
     const {data} = await loadTags('NEVER_RELOAD')
     allTags = Object.fromEntries(data.map((r) => [r.name, r])) as Record<string, Tag>
   }
+  if (Object.keys(allBanners).length === 0) {
+    const {data} = await loadBanners('NEVER_RELOAD')
```

Comment:
this is the only place where it's called? `NEVER_RELOAD` doesn't sound right, normal cache should be used with reloading in background after 10 minutes IMHO 🤔